extensions/2.0/Vendor/EXT_feature_metadata/examples/microcosm/microcosm.gltf

```diff
@@ -0,0 +1,286 @@
+                    }
+                }
+            },
+            "featureCollections": {
+                "LandCover": {
+                    "featureClass": "LULC",
+                    "featureCount": 6,
+                    "properties": {
+                        "Name": {
+                            "elementCount": 256,
+                            "bufferView": 5,
+                            "offsetsBufferView": 6
+                        },
+                        "Color": {
+                            "elementCount": 256,
+                            "bufferView": 7
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "scene" : 0,
+    "scenes" : [
+        {
+            "name" : "Scene",
+            "nodes" : [
+                0,
+                1
+            ]
+        }
+    ],
+    "nodes" : [
+        {
+            "name" : "Camera",
+            "rotation" : [
+                0.483536034822464,
+                0.33687159419059753,
+                -0.20870360732078552,
+                0.7804827094078064
+            ],
+            "translation" : [
+                7.358891487121582,
+                4.958309173583984,
+                6.925790786743164
+            ]
+        },
+        {
+            "mesh" : 0,
+            "name" : "Terrain"
+        }
+    ],
+    "materials" : [
+        {
+            "doubleSided" : true,
+            "name" : "Photogrammetry",
+            "pbrMetallicRoughness" : {
+                "baseColorTexture" : {
+                    "index" : 0,
+                    "texCoord" : 1
+                },
+                "metallicFactor" : 0,
+                "roughnessFactor" : 0.5
+            }
+        }
+    ],
+    "meshes" : [
+        {
+            "name" : "Terrain Mesh",
+            "primitives" : [
+                {
+                    "attributes" : {
+                        "POSITION" : 0,
+                        "NORMAL" : 1,
+                        "TEXCOORD_0" : 2,
+                        "TEXCOORD_1" : 3
+                    },
+                    "indices" : 4,
+                    "material" : 0
+                }
+            ]
+        }
+    ],
+    "textures" : [
+        {
+            "name": "Photogrammetry Texture",
+            "source" : 0
+        },
+        {
+            "name": "NDVI Texture",
+            "source" : 1,
+            "extensions": {
+                "EXT_feature_metadata": {
+                    "features": [
+                        {
+                            "featureClass": "NDVI",
+                            "properties": {
+                                "NDVI": {
+                                    "channels": "r"
+                                }
+                            }
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "name": "Land Cover Texture",
+            "source" : 2,
```

Comment:
I was about to say that Land Cover needs to use a sampler according to `texture.EXT_feature_metadata.featureMapping.featureIds.schema.json`:

> "The channel containing feature IDs. Feature IDs must be whole numbers in the range `[0, elementCount - 1]`, where `elementCount` is the total number of elements in each property array. Texel values must not be normalized. The texture's sampler must use nearest neighbor filtering - i.e. `minFilter` and `magFilter` must be `9728` (`NEAREST`)."

But then I thought it might be better if the spec states that when the client retrieves feature IDs it should ignore the texture's sampler and always do nearest filtering. Then a sampler wouldn't be required.